### PR TITLE
[MINOR][Docs][WIP] Fix Typos

### PR DESCRIPTION
**Pull request by kjmrknsn**
## What changes were proposed in this pull request?
Fix Typos.

## How was this patch tested?
NA

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
It's okay but mind taking another look the documentation and fix other typos while we are here? I'm pretty sure there are more.

**Comment by kjmrknsn** [comment]
OK, I'll check the whole documentation later. Thanks.

File: docs/index.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Example applications are also provided in Python. For example,
     ./bin/spark-submit examples/src/main/python/pi.py 10
 
 Spark also provides an experimental [R API](sparkr.html) since 1.4 (only DataFrames APIs included).
```
Comment (felixcheung) [inline docs/index.md]: tbh, I'm not sure this should be called "an experimental [R API]"

Comment (kjmrknsn) [reply docs/index.md]: Should we remove the "experimental" adjective?

**Comment by HyukjinKwon** [comment]
ok to test

**Comment by HyukjinKwon** [comment]
ok to test

File: docs/index.md (hunk 1 above)
Comment (felixcheung) [reply docs/index.md]: let's remove `experimental`, WDYT?

Comment (kjmrknsn) [reply docs/index.md]: OK, `experimental` has been removed. https://github.com/apache/spark/pull/23145/files#diff-1a523bd9fa0dbf998008b37579210e12

**Comment by kjmrknsn** [comment]
Sorry, this PR is still WIP. I'll continue to check the documentation later. Thanks.

File: docs/structured-streaming-programming-guide.md [hunk 2]
```diff
@@ -1634,7 +1634,7 @@ returned through `Dataset.writeStream()`. You will have to specify one or more o
 
 - *Query name:* Optionally, specify a unique name of the query for identification.
 
-- *Trigger interval:* Optionally, specify the trigger interval. If it is not specified, the system will check for availability of new data as soon as the previous processing has completed. If a trigger time is missed because the previous processing has not completed, then the system will trigger processing immediately.
+- *Trigger interval:* Optionally, specify the trigger interval. If it is not specified, the system will check for availability of new data as soon as the previous processing has been completed. If a trigger time is missed because the previous processing has not been completed, then the system will trigger processing immediately.
```
Comment (srowen) [inline docs/structured-streaming-programming-guide.md]: "has completed" is actually correct, but the passive voice here is grammatical too.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
I think this is fine to merge, this is a good batch of grammar fixes.

**Comment by kjmrknsn** [comment]
Thanks for reviewing and merging.
I've just finished checking the whole documentation.
Here is the complete version of this PR : https://github.com/apache/spark/pull/23185
Thanks.

**Pull request closed without merging**